share/doc/homebrew/Querying-Brew.md

```diff
@@ -24,7 +24,7 @@ From the manpage:
 
 The current schema version is `v1`. Note that fields may be added to the schema as needed without incrementing the schema. Any significant breaking changes will cause a change to the schema version.
 
-The schema itself is not currently documented outside of the code that generates it: [Formula#to_hash](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/formula.rb#L443)
+The schema itself is not currently documented outside of the code that generates it: [Formula#to_hash](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/formula.rb#L443)
```

Comment:
Currently the referenced method is at https://github.com/Homebrew/brew/blob/master/Library/Homebrew/formula.rb#L1230 but that's going to remain a moving target …